archive/test_result.R

```diff
@@ -0,0 +1,18 @@
+#' TODO document with 'tests' documentation
+#' @export
+#' @rdname test
+#' @param message Message to report back if the test throws an error.
+#' @param fn function to execute against the user solution.
+#'   If the test fails, it should throw an error to display the \code{message} provided.
+# test <- function(message, fn) {
```

Comment:
delete this one. I made it. It is not needed anymore

Reply:
docs already deleted